nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/io/marshallers/json/TestAggregateJsonWriter.java

```diff
@@ -56,4 +67,39 @@ public void testFetchedWithNonAdministrator() throws IOException {
         assertTrue(json.contains("{\"key\":\"dc:creator\",\"fetchedKey\":\"dc:creator\",\"docCount\":10}"));
     }
 
+    @Test
+    @WithUser
+    @LogCaptureFeature.FilterOn(loggerClass = AggregateJsonWriter.class, logLevel = "WARN")
+    @ConsoleLogLevelThreshold("ERROR")
```

Comment:
You probably don't need it as we're not expecting a WARN log, right?

Reply:
You right, i removed it on 10.10 and i forget to remove it in master.
thx